pkg/trustedresources/verify.go

```diff
@@ -46,8 +46,18 @@ const (
 // Return an error when no policies are found and trusted-resources-verification-no-match-policy is set to fail,
 // or the resource fails to pass matched enforce verification policy
 // source is from ConfigSource.URI, which will be used to match policy patterns. k8s is used to fetch secret from cluster
-func VerifyTask(ctx context.Context, taskObj v1beta1.TaskObject, k8s kubernetes.Interface, source string, verificationpolicies []*v1alpha1.VerificationPolicy) error {
-	matchedPolicies, err := getMatchedPolicies(taskObj.TaskMetadata().Name, source, verificationpolicies)
+func VerifyTask(ctx context.Context, taskObj *v1beta1.Task, ct *v1beta1.ClusterTask, k8s kubernetes.Interface, source string, verificationpolicies []*v1alpha1.VerificationPolicy) error {
+	// TODO if we can simply convert ClusterTask to Task here for verify?
+	var objectMeta metav1.ObjectMeta
+
+	if taskObj != nil && ct == nil {
+		objectMeta = taskObj.TaskMetadata()
+	}
+	if taskObj == nil && ct != nil {
+		objectMeta = ct.TaskMetadata()
```

Comment:
so to double check, we do not need to include ClusterTask here? 🤔  since it is not included in trustedresources.

Reply:
yes 😂